src/System.Text.Json/src/System/Text/Json/Node/JsonObject.cs

```diff
@@ -378,8 +164,9 @@ public void Add(string propertyName, IEnumerable<JsonNode> propertyValues)
             var jsonArray = new JsonArray();
             foreach (JsonNode value in propertyValues)
             {
-                jsonArray.Add(value);
+                jsonArray.Add(value ?? new JsonNull());
             }
```

Comment:
Isn't there a JsonArray ctor that takes an enumerable of nodes? Why doesn't this method just call that instead of repeating the work?

Reply:
Right. I will remove it anyway.